From de1ca4c80976f64bff3d317e2082a6782b4c54ab Mon Sep 17 00:00:00 2001
From: Peter Seebach <peter.seebach@windriver.com>
Date: Fri, 13 Jun 2014 13:37:05 -0500
Subject: [PATCH 1/1] Use constant initializer for static

strlen(array) isn't a constant expression, even though gcc can sometimes
figure it out at compile time.
---
diff --git a/pseudo_util.c b/pseudo_util.c
index 325cabf..e4e1fc8 100644
--- a/pseudo_util.c
+++ b/pseudo_util.c
@@ -98,7 +98,7 @@ dump_env(char **envp) {
 int
 pseudo_has_unload(char * const *envp) {
 	static const char unload[] = "PSEUDO_UNLOAD";
-	static size_t unload_len = strlen(unload);
+	static size_t unload_len = sizeof(unload) - 1;
 	size_t i = 0;
 
 	/* Is it in the caller environment? */
-- 
2.4.2

